ElectronNET.API/Entities/ProcessVersions.cs

```diff
@@ -0,0 +1,18 @@
+namespace ElectronNET.API
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    public class ProcessVersions
+    {
+        /// <summary>
+        /// Gets or sets a value representing Chrome's version string.
+        /// </summary>
+        public string Chrome { get; set; }
```

Comment:
If these are read only values you might make the set internal for these two props.

Reply:
Alternatively, i figured this would be a perfect use case for using a record type to achieve the desired read-only behavior. See my latest commit 👍